fission/function.go

```diff
@@ -392,6 +392,13 @@ func fnUpdate(c *cli.Context) error {
 
 	envName := c.String("env")
 	envNamespace := c.String("envNamespace")
+	// if the new env specified is the same as the old one, no need to update package
+	// same is true for all update parameters, but, for now, we dont check all of them - because, its ok to
+	// re-write the object with same old values, we just end up getting a new resource version for the object.
+	if envName == function.Spec.Environment.Name && envNamespace == function.Spec.Environment.Namespace {
+		envName = ""
```

Comment:
Since we adopted multi-tenancy, the the namepsace of evnironment may be changed. For example, user create a same name environment obj but in different namespace.
So please set namespace empty here as well.
https://github.com/fission/fission/blob/80fadcc7b871cb410c50b465c40ec0d20f79dd11/fission/package.go#L148-L152
I also noticed there is a tiny bug in `package.go`, we need to check `envName ` and `envNamespace ` separately in case that user only changed the namespace. I am fine to fix this in later PR, but it will be great if we can solve it in this PR as well.